snapshot/snapshot.go

```diff
@@ -716,7 +718,31 @@ func overlayMount(options []string) []mount.Mount {
 		},
 	}
 }
+func (o *snapshotter) passImageNameMounts(ctx context.Context, parent string) ([]mount.Mount, error) {
+	_, pInfo, _, err := snapshot.GetSnapshotInfo(ctx, o.ms, parent)
+	if err != nil {
+		return nil, errors.Wrapf(err, "passImageNameMounts get snapshot %q info failed", parent)
+	}
+	imageURL, ok := pInfo.Labels["containerd.io/snapshot/cri.image-ref"]
 
+	if !ok {
+		return nil, errors.Errorf("passImageNameMounts failed to find image ref of snapshot %s, labels %v",
+			parent, pInfo.Labels)
+	}
+	passImageOption := fmt.Sprintf("imageurl=%s", base64.StdEncoding.EncodeToString([]byte(imageURL)))
```

Comment:
How about follow https://github.com/containerd/containerd/blob/fb786b2f17b2c7d7ea5d9c9009d980bc03379952/pkg/snapshotters/annotations.go#L34, name it `image-ref` ?